<launch>

    <!-- Arguments-->
    <arg name="config_path" value="$(find for_franka_ros)/config/topics.yaml"/>
    
    <!-- Start Gazebo + MoveIt!-->
    <include file="$(find panda_moveit_config)/launch/demo_gazebo.launch"/>

    <!-- Start arm interface -->
    <node pkg="for_franka_ros"  type="for_franka_ros_node" name="control_arm_node" output="screen">
        <param name="config_path" value="$(arg config_path)" />
    </node>

</launch>